tasks.go

```diff
@@ -623,8 +624,18 @@ func TaskWebServe(t *T) error {
 	}
 
 	t.Log("Serving locally...")
-	t.Cd(webDir)
-	return t.Exec("python3 -m http.server 8080")
+	http.Handle("/", http.FileServer(http.Dir("web")))
+	return http.ListenAndServe(":8080", nil)
```

Comment:
Might want to extract this change into its own PR since it doesn't really belong here. Just something a stumbled upon as I was trying to ensure the app was served for testing purposes.

See also  #361